api/CustomElementRegistry.json

```diff
@@ -247,7 +237,17 @@
             "safari_ios": {
               "version_added": "10.1",
               "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
-            }
+            },
+            "webview_android": [
+              {
+                "version_added": "66",
```

Comment:
Shouldn’t this be `67` [like above](https://github.com/mdn/browser-compat-data/pull/1708/files#diff-c0c39ea3ac0b61fea9f2765a6263d4e5R138)?